src/apps/Agreement/ConfigurationChecklist.js

```diff
@@ -0,0 +1,209 @@
+  }, [items])
+
+  return (
+    <Box heading="Configuration checklist" padding={0}>
+      <Transition
+        native
+        items={checklistComplete}
+        from={{
+          opacity: 0,
+          transform: 'scale(0.7)',
+        }}
+        enter={{
+          opacity: 1,
+          transform: 'scale(1)',
+        }}
+        config={springs.smooth}
+      >
+        {show =>
+          show &&
+          (props => (
+            <BoxInner>
+              <AnimatedDiv
+                style={props}
+                css={`
+                  display: flex;
+                  flex-direction: column;
+                  align-items: center;
+                `}
+              >
+                <h2
+                  css={`
+                    text-align: center;
+                    margin-bottom: ${2 * GU}px;
+                  `}
+                >
+                  All done! Congratulations!
+                </h2>
+                <img
+                  src={checklistCompleteGraphic}
+                  css={`
+                    height: 130px;
+                  `}
+                />
+              </AnimatedDiv>
+            </BoxInner>
+          ))
+        }
+      </Transition>
+      {!checklistComplete &&
+        items.map(([label, checked]) => (
+          <BoxInner key={label}>
+            <ChecklistItem label={label} checked={checked} />
+          </BoxInner>
+        ))}
+
+      <BoxInner>
+        <div
+          css={`
+            padding-bottom: ${1 * GU}px;
+          `}
+        >
+          <label
+            css={`
+              ${textStyle('label2')}
+
+              display: flex;
+              justify-content: space-between;
+              color: ${theme.surfaceContentSecondary};
+
+              line-height: 1.2;
+              margin-bottom: ${1 * GU}px;
+            `}
+          >
+            <span>Progress</span>
+            <span>
+              {progress}/{items.length}
+            </span>
+          </label>
+          <ProgressBar value={barProgress} />
+          {checklistComplete && (
+            <Button
+              label="Close"
+              wide
+              onClick={onClose}
+              css={`
+                margin-top: ${2 * GU}px;
+              `}
+            />
+          )}
+        </div>
+      </BoxInner>
+    </Box>
+  )
+}
+
+/* eslint-disable react/prop-types */
+function ChecklistItem({ label, checked }) {
+  const theme = useTheme()
+
+  return (
+    <label
+      css={`
+        display: flex;
+        align-items: center;
+      `}
+    >
+      <div
+        css={`
+          display: flex;
+          align-items: center;
+          justify-content: center;
+          width: ${3 * GU}px;
+          height: ${3 * GU}px;
+          border-radius: 100%;
+          margin-right: ${1.5 * GU}px;
+          background-color: ${theme.infoSurface};
+        `}
+      >
+        {checked && <Check color={theme.surfaceContent} />}
```

Comment:
This presentation is different to a the checkbox as it's a non-interactive element and thus should not:
- Look like other elements that are known to be intractable 
- Not have or inherit hover, active or focus states

Reply:
Might be good to lift this as a comment to explain why we're not using a `Checkbox`!